app/controllers/user_filters_controller.rb

```diff
@@ -13,10 +13,12 @@ def create
     redirect_to root_url unless current_user.user_filters.count < 100 # Enforce upper limit per user
     active_filter_options = params.require(:activeFilterOptions).collect do |filter|
       {
-        filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, options: []),
+        filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :allowRange, options: []),
```

Comment:
Why has `: hasTimestamp` and `allowRange` been added here (and below)? I don't see them being used.

Reply:
These are necessary for when you reload a saved filter (a bug @hackrm pointed out above on this PR)

Reply:
Ah gotcha, understood.